README.md

```diff
@@ -35,6 +35,22 @@ To use N-API in a native module:
 #include "napi.h"
 ```
 
+To make the migration to node-api easier, we have provided a script to help complete the steps listed above. To use the conversion script:
+  1. Go to your module directory
+```
+cd [module_path]
+```
+  2. Install node-api module
+```
+npm install node-api
+```
+  3. Run node-api conversion script
+```
+node ./node_modules/node-api/tools/conversion.js ./
+```
+  4. While this script makes conversion easier, it still cannot fully convert the module.  The next step is to try to build the module and complete the remaining conversions necessary to allow it to compile and pass all of the module's tests.
+
+
 At build time, the N-API back-compat library code will be used only when the targeted node version *does not* have N-API built-in.
```

Comment:
Move this line up before the conversion tool section, because it is related to the .gyp build logic above.